dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloaderUtils.java

```diff
@@ -282,6 +282,20 @@ public static void checkResponseCode(int repoResponseCode, String repo) throws I
         }
     }
 
+    /**
+     * Users may not have access to the entire repository so we don't check other 400 or greater level response codes
+     *
+     * @param repoResponseCode
+     * @return true if responseCode is anything other than 404
+     */
+    public static boolean checkResponseCode(int repoResponseCode) {
+        if (repoResponseCode != 404) {
+            return true;
+        }
+        //TODO add finest logging
```

Comment:
complete the TODO here, maybe you're waiting logging changes discussed previously :) . Those likely wont get in before you merge so just remove the TODO. 

In general,  I prefer to have one return in a given method. it helps whenever debugging. I'll eave that up to you to decide though

Reply:
For clarity I would check if it's 404 and return false otherwise true. just easier to read imo. Again, a nit. 
if (repoResponseCode == 404)